plugin-server/src/kafka/batch-consumer.ts

```diff
@@ -218,26 +248,30 @@ export const startBatchConsumer = async ({
                 }
 
                 const startProcessingTimeMs = new Date().valueOf()
+                const batchSummary = new BatchSummary()
 
                 consumerBatchSize.labels({ topic, groupId }).observe(messages.length)
                 for (const message of messages) {
                     consumedMessageSizeBytes.labels({ topic, groupId }).observe(message.size)
+                    batchSummary.record(message)
                 }
 
                 // NOTE: we do not handle any retries. This should be handled by
                 // the implementation of `eachBatch`.
+                status.info('⏳', `Starting to process batch of ${messages.length} events...`, batchSummary)
```

Comment:
We currently log at `info` in prod, I think this will be too noisy? I think this and the one below should be `debug`?

Reply:
Hmm, yeah… I was on the fence here, but thinking about it more, I guess this mostly matters in the slow batch case and the extra context would be available in that situation since it logs at warning (as long as the pod doesn't get killed) so downlevelling it makes sense.

Reply:
Yeah I think in the happy path we process lots of tiny batches, I remember adding logging once and it was a wall of spam.